src/streamlink/plugins/nrk.py

```diff
@@ -3,30 +3,10 @@
 
 from streamlink.plugin import Plugin
 from streamlink.plugin.api import validate
-from streamlink.stream import HLSStream
+from streamlink.stream import HLSStream, HTTPStream
 
-COOKIE_PARAMS = (
-    "devicetype=desktop&"
-    "preferred-player-odm=hlslink&"
-    "preferred-player-live=hlslink"
-)
-
-_id_re = re.compile(r"/(?:program|direkte|serie/[^/]+)/([^/]+)")
+_id_re = re.compile(r"/(program|direkte|serie|podkast)(?:/.+)?/([^/]+)")
```

Comment:
It's better to have the URL regex include the ID regex, so that there's only a single URL to match the input. Not sure why it's been done this way previously, but if the line is too long, use the verbose regex option.